How-To-Fork-And-Maintain-a-Local-Instance-of-Free-Code-Camp.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+If you plan to write a Pull Request for Free Code Camp, you will almost certainly need a local copy of the site. Having a local copy of the site will give you additional capability with Git that are not availible via the GitHub browser interface, including updating your fork and rebasing/squashing commits.
+
+This guide will cover how to fork the FCC project, clone a local copy, and how to maintain your fork. All Git commands will be given for the command line, which we strongly reccomend that you use, but all commands can be executed in a graphical Git enviornment as well.
```

Comment:
I wouldn't say that all commands can be executed in a graphical Git environment, only most of them.